src/main/twirl/uk/gov/hmrc/govukfrontend/views/layouts/twoThirdsOneThirdMainContent.scala.html

```diff
@@ -0,0 +1,13 @@
+@this()
+
+@(mainContent: Html, sidebarContent: Html)
```

Comment:
Curious if `@(sidebarContent: Html)(mainContent: Html)` would mean you could then do `mainContentLayout = Some(twoThirdsOneThirdMainContent(sidebar))`?

Reply:
@matthewmascord Yes exactly as you suggest, have updated.